src/mongoc/mongoc-cursor-private.h

```diff
@@ -46,7 +46,13 @@ struct _mongoc_cursor_interface_t
    void             (*get_host) (mongoc_cursor_t        *cursor,
                                  mongoc_host_list_t     *host);
 };
+#ifdef MIN
+#undef MIN
+#endif
 
+#ifdef MAX
+#undef MAX
+#endif
```

Comment:
sorry, yeah. I mean, we should rename our MAX macro there to something else.

MAX and MIN are quite common macros which we shouldn't be imposing on.
we should probably rename our macros to MAX_STRING, and MAX_STRING_LEN maybe?